### ISSUE TITLE
Latest version seems to trigger account notifications even if they are disabled

### ISSUE THREAD

**Issue by evgenyboxer** [blocknative/notify#189]
Hey @aaronbarnardsound!

I noticed that after upgrading to latest I am receiving account messages even though I have returned `false`. This doesn't happen for `notify.hash` (as expected).

```
      const { emitter } = notify.account(someAccount);
      emitter.on('all', transaction => {
        return false; // latest version seem to ignore this?
      });
```

I was looking at the commits and I couldn't tell what caused this behaviour...  (Previously I was using `v1.5.1`)

**Comment by taylorjdawson** [comment]
@evgenyboxer Hey thanks for bringing this to our attention! I'll take a look at this and get back you

**Comment by evgenyboxer** [comment]
Hey @taylorjdawson any update? thx

**Comment by taylorjdawson** [comment]
Hey @evgenyboxer I haven't had a chance to circle back to this, although towards the end of this week I should be able to look at it. In the meantime could you provide me with the version that you switched from e.g. the last working version?

It is possible that the latest SDK (v3.3.6) could resolve this -- I plan to update that this week

**Comment by taylorjdawson** [comment]
@evgenyboxer [Fix](https://github.com/blocknative/sdk/pull/126) incoming! I'll update and close this issue once we release the updated blocknative sdk

**Comment by evgenyboxer** [comment]
awesome!

**Comment by taylorjdawson** [comment]
@evgenyboxer Fixed! 🙌🏾  Update to [v1.6.1](https://github.com/blocknative/notify/releases/tag/1.6.1) and it should work
